stdlib/3/subprocess.pyi

```diff
@@ -808,96 +808,347 @@ class CalledProcessError(Exception):
                  output: Optional[_TXT] = ...,
                  stderr: Optional[_TXT] = ...) -> None: ...
 
-class Popen:
+class _Popen(Generic[AnyStr]):
```

Comment:
Actually now I am not sure underscore+alias is actually needed (and was probably not needed for `CompletedProcess`) since these types will appear in error messages and may confuse people. But let's see what other people think @JelleZijlstra @srittau

Reply:
I'm going to drop it